extensions/ui-extension/src/targets/purchase/thank-you/block/render.tsx

```diff
@@ -1,14 +1,58 @@
 import { Community, Newsletter, GuideLinks } from '@/components/post-purchase';
 import { reactExtensionWithSentry } from '@/helpers/extension';
-import { BlockStack } from '@shopify/ui-extensions-react/checkout';
+import { isHeadlessStore } from '@/helpers/shop';
+import { useIfixitSettings } from '@/hooks/useIfixitSettings';
+import {
+   BlockStack,
+   useAttributes,
+   useShop,
+} from '@shopify/ui-extensions-react/checkout';
+import { Shop } from '@shopify/ui-extensions/checkout';
+import { useEffect } from 'react';
+import * as Sentry from '@sentry/browser';
 
 export default reactExtensionWithSentry(
    'purchase.thank-you.block.render',
-   () => (
+   () => <ThankYou />
+);
+
+function ThankYou() {
+   const attributes = useAttributes();
+   const publicCartid = attributes?.find(
+      (attr) => attr.key === 'publicCartid'
+   )?.value;
+   const shop = useShop();
+   const { ifixitBaseUrlDevOverride } = useIfixitSettings();
```

Comment:
`ifixitBaseUrlDevOverride` is either a dev override or null. We need to use `useIfixitBaseUrl`.

```suggestion
   const ifixitBaseUrl = useIfixitBaseUrl();
```